src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp

```diff
@@ -36,11 +48,36 @@
         for (var i = 0; i < jsonData.length; i++){
             ossData.push(jsonData[i]['oss']);
         }
-        console.log(ossData);
+    }
+
+    function dataValidCheck(mainData) {
+        mainData.forEach((row, index) => {
+            //스트링 데이터 검사
+            stringDataForValid.forEach((strData) => {
+                ossData[index][strData] = row[strData].trim();
+            })
+
+            //리스트 데이터 검사
+            listDataForValid.forEach((listData) => {
+                ossData[index][listData] = row[listData].trim()
+                if (ossData[index][listData] == "") {
+                    ossData[index][listData] = []
+                } else {
+                    ossData[index][listData] = ossData[index][listData].split(",")
+                }
+            })
+        })}
```

Comment:
Indentation of 69 line doesn't look good.
It should be matched with `function dataValidCheck(mainData) {`

Reply:
thank you for your review! I fixed commit right before